src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -7,7 +7,8 @@
 from __future__ import print_function as _
 from __future__ import division as _
 from __future__ import absolute_import as _
-import tensorflow as _tf
+import tensorflow.compat.v1 as _tf
+_tf.disable_v2_behavior()
```

Comment:
Could we move this line outside of the `import`s?

Is this function lightweight? Should it just be included in the functions below?

Reply:
Yes